app/main.py

```
@@ -86,10 +96,36 @@ async def chat(req: ChatRequest, request: Request):
+                    access_token=token,
+                    product_key=product_key,
+                )
+                api_context = build_eligibility_api_context(eligibility)
+            else:
+                api_context = (
+                    "대출 가능 여부 조회 질문이지만 상품 정보가 없습니다. "
+                    "자기계발 대출, 소비분석 대출, 비상금 대출 중 어떤 상품인지 먼저 확인이 필요합니다."
+                )
+    except HTTPException:
+        raise # 401, 404, 400 등 HTTP 오류는 클라이언트에 전달하기 위해 그대로 다시 발생시킴
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)